src/site/content/en/lighthouse-best-practices/no-vulnerable-libraries/index.md

```diff
@@ -2,9 +2,11 @@
 layout: post
 title: Includes front-end JavaScript libraries with known security vulnerabilities
 description: |
-  Learn about `no-vulnerable-libraries` audit.
+  Learn how to make your page more secure by replacing JavaScript libraries
+  with known vulnerabilities.
```

Comment:
This sentence sounds funny to me. It sounds like we're saying that readers should replace `JavaScript libraries` with `known vulnerabilities`. But I can't think of a way to rephrase it.

Reply:
Oh, good catch. I tried a fix. See what you think.